cmd/talosctl/cmd/mgmt/gen/config.go

```diff
@@ -114,10 +90,10 @@ func fixControlPlaneEndpoint(u *url.URL) *url.URL {
 	return u
 }
 
-// GenV1Alpha1Config generates the Talos config bundle
+// V1Alpha1Config generates the Talos config bundle
 //
-// GenV1Alpha1Config is useful for integration with external tooling options.
-func GenV1Alpha1Config(genOptions []generate.GenOption,
+// V1Alpha1Config is useful for integration with external tooling options.
+func V1Alpha1Config(genOptions []generate.GenOption,
```

Comment:
please do something like `var GenV1Alpha1Config = V1Alpha1Config`. 

someone was using it in their project, so I don't want to break them if we rename

Reply:
Done in the parent package.